.circleci/config/workflows/website.yml

```diff
@@ -5,3 +5,10 @@ jobs:
         branches:
           only:
             - master
+
+  - algolia_index:
+      context: static-sites
+      filters:
+        branches:
+          only:
+            - master
```

Comment:
I was going to use `stable-website` here, but I *think* `master` is appropriate

Reply:
`master` may not be deployed to the site for quite a while. It feels like it isn't safe to index on something that isn't live.

Reply:
This is going to have to be `stable-website`, like @tgross mentioned, content on `master` isn't necessarily production ready.

Reply:
Sounds good, will update